src/init.cpp

```diff
@@ -383,10 +385,12 @@ int initApp(char* title, int fullscreen)
 			if ( c == 0 )
 			{
 				printlog("model 0 cannot be NULL!\n");
+				fclose(fp);
 				return 12;
 			}
 		}
 	}
+	fclose(fp);
```

Comment:
This one is redundant and causes a crash, fp is freed several lines lower. One of them must go:
```
	fclose(fp);
	if ( !softwaremode )
	{
		generatePolyModels(0, nummodels);
	}
	fclose(fp);
```